fastly/block_fastly_service_v1_director_test.go

```diff
@@ -253,12 +276,24 @@ resource "fastly_service_v1" "foo" {
     name    = "origin old"
   }
 
+  backend {
```

Comment:
I think I understand these tests.

1. You create a service with 2 backends, each associated with a separate director.
2. You update update the field `weight` on a backend and ensure that it is still associated with the original directory